config/routes.rb

```diff
@@ -79,174 +80,12 @@
-          resources :html_variants, only: %i[index edit update new create show destroy]
-          resources :navigation_links, only: %i[index update create destroy]
-          resources :pages, only: %i[index new create edit update destroy]
-
-          # NOTE: @citizen428 The next two resources have a temporary constraint
-          # while profile generalization is still WIP
-          constraints(->(_request) { FeatureFlag.enabled?(:profile_admin) }) do
-            resources :profile_field_groups, only: %i[update create destroy]
-            resources :profile_fields, only: %i[index update create destroy]
-          end
-        end
-
-        scope path: :moderation, as: "moderation" do
-          resources :reports, only: %i[index show], controller: "feedback_messages" do
-            collection do
-              post "send_email"
-              post "create_note"
-              post "save_status"
-            end
-          end
-          resources :mods, only: %i[index update]
-          resources :moderator_actions, only: %i[index]
-          resources :privileged_reactions, only: %i[index]
-        end
-
-        scope path: :advanced, as: "advanced" do
-          resources :broadcasts
-          resources :response_templates, only: %i[index new edit create update destroy]
-          resources :secrets, only: %i[index]
-          put "secrets", to: "secrets#update"
-          resources :sponsorships, only: %i[index edit update new create destroy]
-          resources :tools, only: %i[index create] do
-            collection do
-              post "bust_cache"
-            end
-          end
-          resources :webhook_endpoints, only: :index
-
-          # We do not expose the Data Update Scripts to all Forems by default.
-          constraints(->(_request) { FeatureFlag.enabled?(:data_update_scripts) }) do
-            resources :data_update_scripts, only: %i[index show] do
-              member do
-                post :force_run
-              end
-            end
-          end
-        end
-
-        scope path: :apps do
-          resources :chat_channels, only: %i[index create update destroy] do
-            member do
-              delete :remove_user
-            end
-          end
-          resources :events, only: %i[index create update new edit]
-          resources :listings, only: %i[index edit update destroy]
-          resources :listing_categories, only: %i[index edit update new create
-                                                  destroy], path: "listings/categories"
-          resources :welcome, only: %i[index create]
-        end
-      end
+      # NOTE: [@ridhwana] This is a temporary condition.
+      # It will require the rails app to be reloaded when the feature flag is toggled
+      # You can find more details on why we had to implement it this way in this PR
+      # https://github.com/forem/forem/pull/13114
+      admin_routes = FeatureFlag.enabled?(:admin_restructure) ? :admin : :current_admin
```

Comment:
Used rails 6.1 draw routes to maintain separate files of the current admin routes and the new admin routes. It will make it easier to delete the old routes post testing.